force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -294,43 +388,68 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Person {
         // Declare private two instance variables - (TEXT) name and (Number) age
-        
+        private String name;
+        private Integer age;
+       
         // Constructor to initialize the name and age
-    
-        // Method introduceYourself that returns the details of the person        
-        // Return a string in the format "Hello, my name is <name> and I am <age> years old."
-        // For example, introduceYourself() should return a string like "Hello, my name is John Doe and I am 28 years old."
-
-        // Getter method that returns the name of the person
-
-        // Getter method that returns the age of the person
-
-        // Setter method that sets the name of the person
-
-        // Setter method that sets the age of the person
+        public Person(String name, Integer age) {
+            this.name = name;
+            this.age = age;
+        }
+        // Method introduceYourself that returns the details of the person       
+        public String introduceYourself(){
+            String personDetails = 'Hello, my name is ' + this.name + ' and I am ' + this.age + ' years old.';
+            return personDetails; // Return a string in the format "Hello, my name is <name> and I am <age> years old."
+        }
+ 
+ 
+        public String getName(){ // Getter method that returns the name of the person
+            return this.name;       
+        }
+ 
+ 
+        public Integer getAge(){ // Getter method that returns the age of the person
+            return this.age;
+        }
+ 
+ 
+        public void setName(String newName){ // Setter method that sets the name of the person
+            this.name = newName;
+        }
+       
+        public void setAge(Integer newAge){ // Setter method that sets the age of the person
+            this.age = newAge;
+        }
     }
-
+ 
+ 
     /**
      * Question 15
      * Create a Person object and returns the details of the person.
      * For example, personDetails() should return a list like ["Hello, my name is John Doe and I am 28 years old.", "John Doe", "28"].
      * @return A string containing the details of the person.
      */
-
+ 
+ 
     public static List<String> personDetails() {
         List<String> results = new List<String>();
-        // Create a new instance of Person class
-        // Name the person 'John Doe' and set the age to 28
-
-        // Add the details of the person using the introduceYourself method
-
-        // Get the name of the person using the getter method and add it to the list
-
+        Person newPerson = new Person('John Doe', 28); // Create a new instance of Person class
+     
+        String newPersonAge = String.valueOf(newPerson.getAge());
```

Comment:
This works just fine, but a shorter way to convert an Integer into a String is to use the **[format()](https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_integer.htm#apex_System_Integer_format)** method:
```suggestion
        String newPersonAge = newPerson.getAge().format();
```